pages/index.js

```diff
@@ -1,9 +1,32 @@
 import React from 'react';
+import Link from 'next/link';
+
+import Layout from '../components/Layout';
 
 export default function Index() {
   return (
-    <div>
-      <p>Clap City Cinema</p>
-    </div>
+    <Layout>
+      <div className="home-page">
+        <Link href="/NowShowing">
```

Comment:
Maybe we should switch to a route name that is all lower case and uses dashes like `now-showing`? We should ask Linkon about this too.